src/topology/local_at_target.lean

```diff
@@ -0,0 +1,133 @@
+/-
+Copyright (c) 2022 Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Andrew Yang
+-/
+import topology.maps
+import topology.sets.opens
+import topology.metric_space.basic
+
+/-!
+# Properties of maps that are local at the target.
+
+We show that the following properties of continuous maps are local at the target :
+- `inducing`
+- `embedding`
+- `open_embedding`
+- `closed_embedding`
+
+-/
+
+open topological_space
+
+variables {α β : Type*} [topological_space α] [topological_space β] (f : α → β) (h : continuous f)
```

Comment:
```suggestion
variables {α β : Type*} [topological_space α] [topological_space β] {f : α → β}
```
I think all lemmas have some hypothesis on `f`, so it can be implicit. Also, I think it's cleaner to just explicitly write `(h : continuous f)` for each lemma that uses it